### PR TITLE
Added https://ihavenotv.com

### PR DESCRIPTION
**Pull request by ArjixWasTaken**

### CODE REVIEW

**Comment by ArjixWasTaken** [comment]
Uhh, the `search` and `getMainPage` functions may look like an overkill
I made them that way in order to filter the results into series and individual episodes

File: app/src/main/java/com/lagradost/cloudstream3/movieproviders/iHaveNoTvProvider.kt [hunk 1]
```diff
@@ -0,0 +1,204 @@
...
+                    title,
+                    href,
+                    this.name,
+                    if (href.contains("/series/")) TvType.TvSeries else TvType.Movie,
+                    poster,
+                    year,
+                    null
+                )
+            }
+            items.add(HomePageList(cat, ArrayList(searchResults.values).subList(0, 5))) // just 5 results per category, app crashes when they are too many
```
Comment (LagradOst) [inline app/src/main/java/com/lagradost/cloudstream3/movieproviders/iHaveNoTvProvider.kt:64]: Wait really? Is this a problem with ram or my bad programming?

Comment (ArjixWasTaken) [reply app/src/main/java/com/lagradost/cloudstream3/movieproviders/iHaveNoTvProvider.kt:64]: dunno
each category had like 80 results
so about 240 items

Comment (ArjixWasTaken) [reply app/src/main/java/com/lagradost/cloudstream3/movieproviders/iHaveNoTvProvider.kt:64]: probably ram, it did look like it loaded, but when it started loading the poster images in the roulette ontop it crashed

**Comment by ArjixWasTaken** [comment]
The build succeeded and from my testing it looks ok.

**Comment by ArjixWasTaken** [comment]
I'll let you decide if you want it merged or not.
We probably need a new ShowType for documentaries

**Comment by LagradOst** [comment]
> I'll let you decide if you want it merged or not. We probably need a new ShowType for documentaries

Hmm, I can add that.

**Pull request merged**